frontend/src/components/button/FloatingButton.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from '@emotion/styled';
+
+import theme from '@/styles/theme';
+
+type Size = 'small' | 'medium' | 'extends';
+
+type Color = 'yellow' | 'green' | 'subGreen';
+
+interface Props extends React.HTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  size?: Size;
+  color?: Color;
+  'aria-label'?: string;
+}
+
+const FloatingButton = ({
+  children,
+  size = 'medium',
+  color = 'yellow',
+  'aria-label': ariaLabel = 'add',
+  ...rest
+}: Props) => {
+  return (
+    <S.Button size={size} color={color} aria-label={ariaLabel} {...rest}>
+      {children}
+    </S.Button>
+  );
+};
+
+export default FloatingButton;
+
+const sizeStyle = (size: Size) => {
+  switch (size) {
+    case 'small':
+      return { width: '40px', height: '40px', fontSize: '12px' };
+    case 'medium':
+      return { width: '60px', height: '60px', fontSize: '16px' };
+    case 'extends':
+      return { width: '100px', height: '40px', fontSize: '16px' };
+  }
+};
```

Comment:
```suggestion
const sizeStyle ={
   small: { width: '40px', height: '40px', fontSize: '12px' },
   medium: { width: '60px', height: '60px', fontSize: '16px' },
   extends: { width: '100px', height: '40px', fontSize: '16px' },
};
```

switch-case보다는 객체를 사용하면 좀 더 명료할 것 같습니다.